packages/subnav/index.js

```diff
@@ -125,6 +129,7 @@ function Subnav({ className, ...restProps }) {
         product={product}
         hasOverflow={hasOverflow}
         isSticky={isSticky}
+        theme={theme}
```

Comment:
SOME of the theming, namely just the theme passed to `button`'s `theme.background`, has to be passed through the `CtaLinks` component. This is a part that doesn't feel ideal long term (Context probably better?) but felt sustainable and reasonable given the scope & intent of this PR.